doc/api/process.md

```diff
@@ -2309,6 +2309,227 @@ import { loadEnvFile } from 'node:process';
 loadEnvFile();
 ```
 
+### Environment variables file parsing specification
+
+This section describes how environment variables file parser reads a file
+and sets up the environment variables in Node.js.
+While `.env` files descend from shell scripts that export environment
+variables, there are important distinctions in how they handle quoting,
+spacing, and escaping values.
+Additionally, the [Dotenv][] package  has played a significant role in
+popularizing this format.
+
+#### Basic parsing
+
+* The parser processes input until it finds a newline, splitting each part into
```

Comment:
Remember to use active sentence style, so wording like "Input data is processed ..." is preferred